examples/apps/presence-tracker/package.json

```diff
@@ -29,6 +29,7 @@
 		"prettier:fix": "prettier --write . --cache --ignore-path ../../../.prettierignore",
 		"start": "webpack serve",
 		"start:client:test": "webpack serve --config webpack.test.cjs",
+		"start:tinylicious": "tinylicious",
```

Comment:
`start:tinylicious` means something different in the examples that have it already, and we already have instructions for starting tinylicious in the README.

I'm not strictly opposed, but if we want to start adding this script it would be good to be uniform across all of our examples and update READMEs to match.

Reply:
I found that the existing stuff is not convenient. Why do we need to go through all that extra work? We can make this easy and not add any tax to the install process.
I think I found places that have start without the appropriate devDep. I don't know about the special cases.